Makefile

```diff
@@ -156,12 +162,18 @@ local-unit-test: ## Run backend tests in the dev environment
 			ci_env=$$(bash <(curl -s https://codecov.io/env)); \
 			CI=true; \
 		fi; \
-		docker-compose $(COMPOSE_OPTS) run --rm -e DEV_MODE_COOKIES= -e CI $$ci_env -T backend bash -c "cd /corpora-data-portal && make container-unittest && if [ "$${CI}" = "true" ]; then apt-get update && apt-get install -y git && bash <(curl -s https://codecov.io/bash) -cF backend,python,unitTest; fi"; \
+		$(MAKE) _unit_test; \
 	else \
 		echo "Running test(s): $(path)"; \
 		docker-compose $(COMPOSE_OPTS) run --rm -e DEV_MODE_COOKIES= -T backend bash -c "cd /corpora-data-portal && python -m unittest $(path)"; \
 	fi
 
+.PHONY: _unit_test
+_unit_test:
+	docker-compose $(COMPOSE_OPTS) run --rm -e DEV_MODE_COOKIES= -e CI $$ci_env -T backend bash -c "cd /corpora-data-portal && make container-unittest && if [ "$${CI}" = "true" ]; then apt-get update && apt-get install -y git && bash <(curl -s https://codecov.io/bash) -cF backend,python,unitTest; fi";
+#	uncomment bellow after https://github.com/chanzuckerberg/corpora-data-portal/issues/1129
```

Comment:
Can you add a `TODO:` to this comment, so tools can recognize it?

Reply:
Look at the processing container, it looks like the latest runs do use the cache (where the cache is the previous built image tagged with the name 'branch-main'), up to the point where the Dockerfile was changed to split up the install_github('cellgeni/sceasy') into its own line. This is better than before, since before it wasn't even using the cache for the BiocManager::install('SingleCellExperiment')  or BiocManager::install('LoomExperiment') lines.

This behavior is expected since the newly-split line is basically forking from the cache. I believe the caching will work properly once this is checked into the main branch; after the first very long build future runs will be able to use the previously built layers generated from this run.

(This is from me uncommenting the commented line and running in my side branch)

I'd recommend uncommenting it, letting the first long run finish and push the new image to main, then seeing if it still takes a long time for future PRs.